src/chart.js

```diff
@@ -252,6 +252,78 @@
         _animateBar(target, 0);
       });
     }
+
+    if (options.type == 'gradient-bar') {
+      return this.each(function() {
+        var target = this;
+
+        $(target).attr('data-chart', '');
+
+        // If the background color was not specified then use the container's one
+        if (_settings.backgroundColor === '') {
+          _settings.backgroundColor = $(target).css('background-color') ;
+        }
+
+        // Set the background and border colors of the chart
+        $(target).css('background-color', _settings.backgroundColor);
+        $(target).css('border-color', _settings.backgroundColor);
+
+        // Set the height of the parent of all bar elements
+        $(target).height(typeof _settings.height === "number" ? _settings.height : $(target).height());
+
+        var barPercentageValue = data[0].value;
+
+        // Inject a div for the bar if its value is > 0
+        if (barPercentageValue > 0) {
+          $(target).append(
+            $(document.createElement('div')).height('100%')
+              .css({
+                /* jshint ignore:start */
+                "background": data[0].color,
+                "background": "-webkit-gradient(linear, right top, left top, from(#FFFFFF), to(" + data[0].color + "))",
```

Comment:
You can omit vendors. Rule is well supported https://www.w3schools.com/css/css3_gradients.asp